gridt/src/app/home/home.page.ts

```diff
@@ -190,7 +192,12 @@ export class HomePage implements OnInit, OnDestroy {
         {
           text: "Yes",
           handler: (data) => {
-            this.signal(movement, data.message);
+            if(data.message !==null && data.message.length < 140){
```

Comment:
Right now, if the user has made given no message, but presses yes, it will simply send. Is this desired behaviour?

Reply:
Actually, !== means the message may not be empty. But if it is empty, we might want a custom error message.